includes/AMP/Sanitization.php

```diff
@@ -385,6 +412,25 @@ protected function get_sanitizers(): array {
 			AMP_Tag_And_Attribute_Sanitizer::class => [],
 		];
 
+		$post = get_queried_object();
+
+		if ( $post instanceof WP_Post && Story_Post_Type::POST_TYPE_SLUG === $post->post_type ) {
```

Comment:
```suggestion
		if ( Story_Post_Type::POST_TYPE_SLUG === get_post_type( $post ) ) {
```

This is a little cleaner.

Reply:
Still need the `instanceof` check, otherwise PHPStan complains.

Then there's not really a huge difference between `if ( $post instanceof WP_Post && Story_Post_Type::POST_TYPE_SLUG === $post->post_type )` and `if ( $post instanceof WP_Post && Story_Post_Type::POST_TYPE_SLUG === get_post_type( $post ) )`...